porcupine/plugins/highlight/tree_sitter_highlighter.py

```diff
@@ -140,9 +140,14 @@ def _decide_tag(self, node: tree_sitter.Node) -> str:
         else:
             type_name = node.type
 
-        config_value = self._config.token_mapping.get(type_name, "Token.Text")
+        if set(type_name) <= set("+-*/%~&|^!?<>=@.,:;()[]{}"):
```

Comment:
The characters are currently same as string.punctuation except that they are missing ``'\`#_"$``. I think that's good.